nautobot/docs/docker/index.md

```diff
@@ -0,0 +1,102 @@
+* `latest` these images are built from the latest code in the main branch (should be the latest released version) based on the default python version (3.6) docker container
+* `X.Y.Z-py${PYTHON_VER}` these images are built with the same baseline as the released Python packages based on the python version ($PYTHON_VER) docker container
+* `latest-py${PYTHON_VER}` these images are built from the latest code in the main branch (should be the latest released version) based on the python version ($PYTHON_VER) docker container
+* `develop` these images are built from the latest code in the develop branch on each commit based on the default python version (3.6) docker container
+* `develop-${GIT_SHA:0:7}-$(date +%s)` tags for each commit to the develop branch based on the default python version (3.6) docker container
+* `develop-latest-py${PYTHON_VER}` these images are built from the latest code in the develop branch on each commit based on the python version ($PYTHON_VER) docker container
+* `develop-${GIT_SHA:0:7}-$(date +%s)-py${PYTHON_VER}` tags for each commit to the develop branch based on the python version ($PYTHON_VER) docker container
+
+Currently images are pushed for the following python versions:
+
+* 3.6
+* 3.7
+* 3.8
+* 3.9 _For Testing Only_
+
+!!! info
+    A dev image `networktocode/nautobot-dev` and `ghcr.io/nautobot/nautobot-dev` is also provided with the same tags, this image provides the development dependencies needed to build Nautobot.  This container can be used as a base for development to develop additional Nautobot plugins but should **NOT** be used in production.
+
+## Getting Started
+
+Nautobot requires a Postgres database and Redis cache before it will start, because of this the quickest and easiest way to get Nautobot running is with docker-compose.  Please see the docker-compose deployment repository for more information.
+
+## Configuration
+
+Most configuration parameters are available via environment variables which can be passed to the container.  If you desire you can inject your own `nautobot_config.py` by overriding `/opt/nautobot/nautobot_config.py`.
+
+### UWSGI
+
+The docker container uses [uWSGI](https://uwsgi-docs.readthedocs.io/) to serve Nautobot.  A default configuration is [provided](/docker/uwsgi.ini), and can be overridden by injecting a new `uwsgi.ini` file at `/opt/nautobot/uwsgi.ini`.  There are a couple of environment variables provided to override some uwsgi defaults:
+
+#### `NAUTOBOT_UWSGI_LISTEN`
+
+Default: `128`
+
+The socket listen queue size of uWSGI.  In production environments it is recommended to increase this value to 1024 or higher, however depending on your host OS, this may require additional kernel parameter settings, please see [the uWSGI documentation](https://uwsgi-docs.readthedocs.io/en/latest/articles/TheArtOfGracefulReloading.html#the-listen-queue) for more information.
```

Comment:
```suggestion
The socket listen queue size of uWSGI.  In production environments it is recommended to increase this value to 1024 or higher, however depending on your platform, this may require additional kernel parameter settings, please see [the uWSGI documentation](https://uwsgi-docs.readthedocs.io/en/latest/articles/TheArtOfGracefulReloading.html#the-listen-queue) for more information.
```